_build/builder/CheckDependencies.py

```diff
@@ -24,7 +24,6 @@
 import sys
 import warnings
 from os.path import basename, exists, expandvars, join
-from pip._vendor import requests
```

Comment:
Didn't tested it but i guess writing just `import requests` would have been sufficent.

Reply:
yes but then we would have had to add requests as part of the dependencies for the build.

Now I know there is a kind of hackish way agithub is added without adding a dependency and requests would work as well. but the right way to go about it is to either use urllib2 (like i did) or add a dependency for requests